themes/default/content/templates/serverless-application/azure/index.md

```diff
@@ -0,0 +1,139 @@
+$ pulumi new serverless-azure-typescript
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```bash
+$ mkdir my-serverless-app && cd my-serverless-app
+$ pulumi new serverless-azure-python
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```bash
+$ mkdir my-serverless-app && cd my-serverless-app
+$ pulumi new serverless-azure-go
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```bash
+$ mkdir my-serverless-app && cd my-serverless-app
+$ pulumi new serverless-azure-csharp
+```
+
+{{% /choosable %}}
+
+Follow the prompts to complete the new-project wizard. When it's done, you'll have a complete Pulumi project that's ready to deploy and configured with the most common settings. Feel free to inspect the code in {{< langfile >}} for a closer look.
+
+## Deploying the project
+
+The template requires no additional configuration. Once the new project is created, you can deploy it immediately with [`pulumi up`]({{< relref "/docs/reference/cli/pulumi_up" >}}):
+
+```bash
+$ pulumi up
+```
+
+When the deployment completes, Pulumi exports the following [stack output]({{< relref "/docs/intro/concepts/stack#outputs" >}}) values:
+
+originURL
+: The HTTP URL of the static website.
+
+apiURL
+: The HTTP URL of the serverless function endpoint.
+
+Output values like these are useful in many ways, most commonly as inputs for other stacks or related cloud resources. The computed `originURL`, for example, can be used from the command line to open the newly deployed website in your favorite web browser:
+
+```bash
+$ open $(pulumi stack output originURL)
+```
+
+## Customizing the project
+
+Projects created with the serverless template expose the following [configuration]({{< relref "/docs/intro/concepts/config" >}}) settings:
+
+www_path
+: The path to the folder containing the files of the website. Defaults to `www`, which is the name (and relative path) of the folder included with the template.
+
+api_path
+: The path to the folder containing the files of the serverless function. Defaults to `api`, which is the name (and relative path) of the folder included with the template.
+
+index_document
+: The file to use for top-level pages. Defaults to `index.html`.
+
+error_document
+: The file to use for error pages. Defaults to `error.html`.
+
+All of these settings are optional and may be adjusted either by editing the stack configuration file directly (by default, `Pulumi.dev.yaml`) or by changing their values with [`pulumi config set`]({{< relref "/docs/reference/cli/pulumi_config_set" >}}) as shown below.
+
+### Using your own web content
+
+If you already have a serverless application you'd like to deploy on Azure with Pulumi, you can do so either by replacing placeholder content in the `www` folder or by configuring the stack to point to another folder on your computer with the `www_path` setting:
+
+```bash
+$ pulumi config set www_path ../my-existing-website/build
+$ pulumi up
+```
```

Comment:
This might be technically _okay_, but it won't really work with your own web content unless you write the front-end JS code to call the serverless function. So it might be better either to exclude this section for now or to add the relevant front-end details.

I'd probably suggest omitting for now; otherwise, the user would change the config, run Pulumi, and then be left wondering what to do next. I'd be happy to write this content for you as a follow-up if you want to file an issue for it, though.

```suggestion
All of these settings are optional and may be adjusted either by editing the stack configuration file directly (by default, `Pulumi.dev.yaml`) or by changing their values with [`pulumi config set`]({{< relref "/docs/reference/cli/pulumi_config_set" >}}).

```bash
$ pulumi config set sitePath ../my-existing-website/build
$ pulumi up
```
```